docs/src/main/sphinx/redirects.txt

```diff
@@ -10,6 +10,7 @@ connector/atop.md connector/removed.md
 connector/localfile.md connector/removed.md
 connector/accumulo.md connector/removed.md
 connector/kinesis.md connector/removed.md
+object-storage/legacy-cos.md object-storage/file-system-s3.md 
```

Comment:
we only have that support via the S3 support now .. and thats better anyway. If we add tests we could add something to the S3 page .. but not without it.